oaknational/Oak-Web-Application
# feat: deprecate InlineSpriteSheet and store logos as components

## Description

- deprecate `InlineSpriteSheet` and `InlineSpriteSvg` components
- store logo as component

## Issue(s)

Fixes [PUPIL-1139](https://www.notion.so/oaknationalacademy/Retire-SVG-component-from-OWA-15826cc4e1b180868a33ef7dd54e3e8b)

## How to test

1. Go to https://deploy-preview-3069--oak-web-application.netlify.thenational.academy
2. Check that there are no visual differences to the logo 
3. Brief regression testing on other pages (should be no differences to any svg assets)

## Checklist
